### PR TITLE
Object3D updates

### PR DESCRIPTION
**Pull request by matthias-ccri**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://threejs.org/docs/#api/core/Object3D
https://github.com/mrdoob/three.js/blob/master/src/core/Object3D.js
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/three/three-core.d.ts
Comment (matthiasdailey-ccri) [inline types/three/three-core.d.ts]: Camera.lookAt inherits from Object3D.
https://threejs.org/docs/#api/cameras/Camera
https://github.com/mrdoob/three.js/blob/master/src/cameras/Camera.js

File: types/three/three-core.d.ts
Comment (matthiasdailey-ccri) [inline types/three/three-core.d.ts]: Removed properties no longer in source.

**Review by dhritzkiv — APPROVED** [review]
Most everything looks good to me. 

The only nitpick I might have is removing the deprecated `shading` and `wrapRGB` properties, as I'm not sure what the policy is on removing deprecations (especially since it's still defined in source, as a deprecation).

I'll let the project maintainers decide whether to merge as-is, or require the deprecations to be left in. (I imagine the maintainers will be okay with it)

**Comment by matthiasdailey-ccri** [comment]
Thanks @dhritzkiv,
I didn't notice `shading` and `wrapRGB` were still defined in [Three.Legacy.js:1169](https://github.com/mrdoob/three.js/blob/b3a7ea1736ab4f4b59229a4c72f82455b6a6a7f5/src/Three.Legacy.js#L1169).
I'm not sure what the purpose of that file is. If needed, I can add those properties back into the typings, but it didn't seem right for those using the normal version, for the typings to include properties that aren't actually there. Maybe there should be a separate typings declaration like `THREE.Legacy.Material` to cover that file, or else people can just use an older version of these typings.

**Comment by andy-ms** [comment]
I think if something isn't officially part of the library, it shouldn't be in the types even if it will exist at runtime.

**Pull request merged**
